bcdoc/restdoc.py

```diff
@@ -138,6 +138,10 @@ def path(self):
     def path(self, value):
         self._path = value
 
+    @property
+    def available_sections(self):
+        return list(self._structure.keys())
```

Comment:
In python3, I was getting a view object back that had some deviations from list behavior. So I just forced to be a list to be consistent across python versions.